src/analysis/calculus/times_cont_diff.lean

```diff
@@ -873,6 +873,22 @@ begin
   exact ⟨by rwa insert_eq_of_mem hx, λ x hx, by simp [ftaylor_series_within]⟩
 end
 
+@[simp] lemma times_cont_diff_within_at_zero (hx : x ∈ s) :
```

Comment:
I don't think this should be a simp lemma, the left hand side looks simpler than the right hand side to me.